### PR TITLE
(php_7.0.x) Added supported old stable package

### PR DESCRIPTION
**Pull request by AdmiringWorm**

### CODE REVIEW

File: automatic/php_7.0.x/tools/chocolateyUninstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
```
Comment (pascalberger) [inline automatic/php_7.0.x/tools/chocolateyUninstall.ps1]: $ErrorActionPreference = 'Stop'?

File: automatic/php_7.0.x/tools/chocolateyUninstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+if ($installLocation) {
+  UninstallPackage -libDirectory "$toolsPath\.." -packageName $packageName
+
+  $di = Get-ChildItem $installLocation -ea 0 | Measure-Object
+  if ($di.Count -eq 0) {
+    Remove-Item -Force -ea 0 $installLocation
+  }
+
+  Uninstall-ChocolateyPath $installLocation
+
```
Comment (pascalberger) [inline automatic/php_7.0.x/tools/chocolateyUninstall.ps1]: Remove empty line

File: automatic/php_7.0.x/update.ps1 [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    @('not-threadsafe'; $Latest.URLNTS32; $Latest.URLNTS64; $Latest.ChecksumNTS32; $Latest.ChecksumNTS64) -join '|'
+  )
+
+  [System.IO.File]::WriteAllLines("$PSScriptRoot\tools\downloadInfo.csv", $lines);
+}
+
+function global:au_SearchReplace {
+   @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(?i)(^\s*packageName\s*=\s*)('.*')"                        = "`$1'$($Latest.PackageName)'"
```
Comment (pascalberger) [inline automatic/php_7.0.x/update.ps1]: Remove leading spaces in front of =

**Comment by AdmiringWorm** [comment]
@pascalberger I believe I've fixed the requested changes

**Pull request merged**